src/ctia/schemas/graphql/resolvers.clj

```diff
@@ -3,6 +3,7 @@
             [ctia.domain.entities
              :refer
              [page-with-long-id un-store un-store-page with-long-id]]
+            [ctia.properties :as p]
```

Comment:
GraphQL resolvers will be changed soon to accept a map of services. For now, using global properties.